subprojects/zlib.wrap

```diff
@@ -1,8 +1,9 @@
 [wrap-file]
 directory = zlib-1.2.13
-source_url = http://zlib.net/fossils/zlib-1.2.13.tar.gz
-source_filename = zlib-1.2.13.tar.gz
-source_hash = b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30
+source_url = http://zlib.net/fossils/zlib-1.2.13.tar.xz
```

Comment:
This does not even exist, why are we changing to it?

Reply:
Ah it was changed. Makes sense.

Reply:
No, this has always been how zlib operates. The current version is available as gzip from a top-level download url, and old versions are available also as gzip in the fossils/ directory.

The current version, and only the current version, may have other download formats and perhaps PGP signatures as well. It used to be that the current version was additionally available in .bz2 and .zip, although currently it is xz instead of bz2.